hack/ci/e2e-conformance.sh

```diff
@@ -281,14 +241,30 @@ create_cluster() {
   # actually create the cluster
   KIND_IS_UP=true
 
+  if [[ -z ${IMAGE_ID:-} ]]; then
+    # default lookup org hould be the same as defaultMachineAMIOwnerID
+    IMAGE_LOOKUP_ORG=${IMAGE_LOOKUP_ORG:-"258751437250"}
+    filter="capa-ami-ubuntu-18.04-1.17.*"
```

Comment:
Would this need to be updated when the minor version of k8s is updated?

Reply:
Not necessarily. The only time this would need to be updated is if there are some packaging or dependency related changes that are no longer compatible with the newer packages we are overriding at bootstrapping time.